source/blog/_posts/2018-09-28-clickstream-analysis-with-server-logs.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Clickstream analysis with server logs
+tags:
+    - marketing
+    - Big data
+    - machine learning
+    - clickstream
+indexed: false
+github_link: blog/_posts/2018-09-28-clickstream-analysis-with-server-logs.md
+
+authors: [rc]
+---
+
+Demographic data like server logs are more valuable than many shop user think. With server logs we identify user groups, push valuable products or do customer specific shop marketing with data lying around on the server. In this blog post I will describe how to use the data in a theoretical way with a book selling online shop.
+
+### Identify the customer
+
+First of all, lets divide the customers in two groups. Traditional customers buy printed books and innovative ones buy other types like audio books. To filter the customers we have to prepare the log data and bring the data together to form sessions. Sessions contain all server log entries from a specific IP address starting with the the first view of the shop until he leaves it. An access to a SuperTracker Database can help to validate and add additional attributes and characteristics to the sessions. After defining the sessions they have to be filtered into useful and useless ones. Useless sessions are caused by bots and customers with only one or two log entries who accidentally visited the shop.
+> (Suchacka & Chodak, 2017)
+
+<div style='text-align:center;'>
+<img src="/blog/img/2018-09-28-clickstream-analysis-with-server-logs/framework.jpg"/>
+
+###### Association rule generation framework for a B2C website (Suchacka & Chodak, 2017)
+</div>
```

Comment:
das schließende DIV muss doch in Zeile 23 oder?

Reply:
muss darunter bleiben sonst ist es nicht zentriert, wie das Bild